src/instruments/view/instrumentpaneltreemodel.cpp

```diff
@@ -61,13 +61,20 @@ InstrumentPanelTreeModel::InstrumentPanelTreeModel(QObject* parent)
 
         emit selectionChanged();
     });
+
+    dispatcher()->reg(this, "instruments", this, &InstrumentPanelTreeModel::addInstruments);
 }
 
 InstrumentPanelTreeModel::~InstrumentPanelTreeModel()
 {
     deleteItems();
 }
 
+bool InstrumentPanelTreeModel::canReceiveAction(const actions::ActionName&) const
+{
+    return true;
```

Comment:
this is the default behavior, no effect from overriding

Reply:
This is outdated changes, in the last version implementation of this method is different